### DISCUSSION TITLE
Separate Flux CD CRDs from application CRDs

### DISCUSSION THREAD

**Discussion by xAdRx** [fluxcd/flux2#4714] [Q&A]
Hey,

Is it possbile to separate CRD-related manifests for FluxCD helm chart and actual component-related manifest

When I had to re-create flux components (after unsuccessful update), it also rebuilt the CRDs for deployed application in my kubernetes cluster. I would like to avoid such situation in future.

Any ideas on how to achieve that?

**Comment by bfennane** [comment]
Sorry to tell you that the only way right now would be to get the CRDs directly from an installation file or the fluxv2 repo.
Apply them using kubectl apply or a terraform provider for example.
ANd then install the helm chart using the flag:
`installCRDs: false`
